Zombie/src/main/java/dk/gruppeseks/bodtrd/zombie/ZombiePlugin.java

```diff
@@ -41,29 +41,27 @@
     @Override
     public void start(World world)
     {
+
         Installer.Plugin = this;
 
         ViewManager.createView(ZOMBIE_IMAGE_FILE_PATH, false);
 
         _world = world;
         _zombies = new HashMap();
 
-        for (int i = 0; i < 10; i++)
+        for (int i = 0; i < 2; i++)
         {
             Entity zombie = createZombieEntity();
             _zombies.put(zombie.getID(), zombie);
             world.addEntity(zombie);
         }
 
-        _processor = new ZombieProcessor(_zombies);
+        _ai = _lookup.lookup(AISPI.class);
```

Comment:
Projectwise i don't think its required to be able to unload/load more than 2, but it still would be interesting to show that their AI changes on runtime.